packages/engine/src/datastore/arrow/field_conversion.rs

```diff
@@ -18,17 +18,17 @@ use crate::datastore::{
 impl PresetFieldType {
     fn is_fixed_size(&self) -> bool {
         match self {
-            PresetFieldType::UInt32 => true,
-            PresetFieldType::UInt16 => true,
+            PresetFieldType::Uint32 => true,
```

Comment:
I'm unsure if `UInt` should be kept, because it comes from `UnsignedInt`. No strong opinion here, though. On the other hand, it's `std::os::raw::c_uint`, not `std::os::raw::c_u_int`.

Reply:
Yeah so the Rust docs specifically say:
> use Uuid rather than UUID, Usize rather than USize

Reply:
I can't say I'm a fan, and it differs from Arrow which are the types we're mirroring here, but I figure we should stay consistent

Reply:
For some types it looks weird, but, yes, let's stay consistent.